### ISSUE TITLE
Infobox problems

### ISSUE THREAD

**Issue by ugurdogrusoz** [iVis-at-Bilkent/newt#331]
- Tiling infoboxes do not refresh the node and its infoboxes (need to select node for an update)
- Undo doesn't seem to work when relocating or tiling infoboxes

**Comment by kaansancak** [comment]
@ugurdogrusoz  refresh problem is solved.

**Comment by kaansancak** [comment]
@ugurdogrusoz  non-uniform margins also should be solved via https://github.com/iVis-at-Bilkent/chise.js/commit/0aa35fe953000a406ce267cb399272a21d13f85c

**Comment by kaansancak** [comment]
@ugurdogrusoz Problems should be fixed now.

**Comment by ugurdogrusoz** [comment]
- create a macromolecule
- create an infobox
- undo
- redo (get an error)

- create a macromolecule
- create an infobox
- label the infobox
- delete the infobox
- undo (get an error)

**Comment by kaansancak** [comment]
@ugurdogrusoz problems mentioned in this issue should be fixed.
